core/app/(user)/login/page.tsx

```diff
@@ -27,9 +30,13 @@ export default async function Login({
 		redirect("/settings");
 	}
 
+	const { notice, error, body } = await searchParams;
+
 	return (
 		<div className="mx-auto max-w-sm">
 			<LoginForm />
+			{notice && <Notice type="notice" title={notice} body={body} />}
+			{error && <Notice type="error" title={error} body={body} />}
```

Comment:
this is also a new addition: the login screen now can display notices. by default, if redirected to here bc of `getPageLoginData`, itll display an error saying "you need to be signed in to access that page". i added this bc i need the user to sign in to their account if its an invite for an existing account, and i think its nice to show why you need to do something on the login page, rather than just being shown a blank one